modules/hardware/nvidia-jetson-orin.nix

```diff
@@ -10,8 +10,32 @@
 
   nixpkgs.hostPlatform.system = "aarch64-linux";
 
-  boot.loader = {
-    efi.canTouchEfiVariables = true;
-    systemd-boot.enable = true;
+  boot.kernelPatches = [
+    {
+      name = "passthrough-patch";
+      patch = ./pci-passthrough-test.patch;
+    }
+  ];
+
+  hardware.deviceTree = {
+    enable = true;
+    name = "tegra234-p3701-host-passthrough.dtb";
+  };
+
+  # Passthrough Jetson Orin WiFi card
+  boot.kernelParams = [
+    "vfio-pci" "ids=10ec:c82f"
+    "vfio_iommu_type1.allow_unsafe_interrupts=1"
+  ];
+
+  boot.loader.efi.canTouchEfiVariables = true;
+  boot.loader.systemd-boot = {
+    enable = true;
+    # TODO: Maybe add store path or some unique identifier to the filename
+    extraFiles."dtbs/${config.hardware.deviceTree.name}" = "${config.hardware.deviceTree.package}/${config.hardware.deviceTree.name}";
+    extraInstallCommands = ''
+      default_cfg=$(cat /boot/loader/loader.conf | grep default | awk '{print $2}')
```

Comment:
This stuff is now available in module boot/systemd-boot-dtb.nix. This whole WIP commit can be removed from this branch

Reply:
Would it make sense to push @panufi 's PR branch from his personal fork to under ghaf? We could then work many of these issues together in the same branch while he is out. Including rebasing with main to get some of the microvm.nix upstream changes merged to main earlier.